nireports/tests/__init__.py

```diff
@@ -0,0 +1,29 @@
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+#
+# Copyright 2023 The NiPreps Developers <nipreps@gmail.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# We support and encourage derived works from this project, please read
+# about our expectations at
+#
+#     https://www.nipreps.org/community/licensing/
+#
+# STATEMENT OF CHANGES: This file was ported carrying over full git history from niworkflows,
+# another NiPreps project licensed under the Apache-2.0 terms, and has been changed since.
+
+from shutil import which
+
+have_compression = which("svgo") is None or which("cwebp") is None
```

Comment:
This returns true if either one are not found.
```suggestion
have_compression = which("svgo") and which("cwebp")
```

Reply:
:+1: using the flag changes the logic for skipping the test. Changed in commit f94343c. My bad. Thanks.